.azure-pipelines/bazel.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+- name: ciTarget
+  displayName: "CI target"
+  type: string
+  default: build 
+
+steps:
+- bash: |
+    echo "disk space at beginning of build:"
+    df -h
+  displayName: "Check disk space at beginning"
+
+- script: ci/run_envoy_docker.sh 'ci/do_ci.sh ${{ parameters.ciTarget }}'
+  workingDirectory: $(Build.SourcesDirectory)
+  displayName: "Run the CI script"
+  env:
+    BAZEL_REMOTE_CACHE: $(LocalBuildCache)
+    AZP_BRANCH: $(Build.SourceBranch)
+    AZP_SHA1: $(Build.SourceVersion)
+    DOCKERHUB_USERNAME: $(DockerUsername)
+    DOCKERHUB_PASSWORD: $(DockerPassword)
```

Comment:
Do we need to add above environment variables in repo settings? Can you help me add them, because I don't have the permission?

Reply:
These variables will be set upstream in AZP.

I suspect that we don't need most of them. Can we remove all and only keep the `BAZEL_REMOTE_CACHE` ?

Reply:
For sure, I will remove them.